src/main/service/SubscriptionService.ts

```diff
@@ -493,11 +508,7 @@ export class SubscriptionService {
                 listType.jurisdictions.some(jurisdiction => courtJurisdictions.includes(jurisdiction)) &&
                 (listType.restrictedProvenances.length === 0 || listType.restrictedProvenances.includes(userRole))
             ) {
-                if (
-                    selectedListTypes == null ||
-                    selectedListTypes.length == 0 ||
-                    selectedListTypes.includes(listName)
-                ) {
+                if (selectedListTypes != null && selectedListTypes.includes(listName)) {
```

Comment:
Do not need an if/else here. Can just do: 

listType.checked  = selectedListTypes != null && selectedListTypes.includes(listName);

Reply:
fixed.